src/components/shortcuts.js

```diff
@@ -293,7 +293,7 @@ import toast from './toast/toast';
         const apiClient = ServerConnections.getApiClient(serverId);
 
         return new Promise((resolve, reject) => {
-            const serverId = apiClient.serverInfo().Id;
+            serverId = apiClient.serverInfo().Id;
```

Comment:
Same here: Overwriting the variable in the outer scope does not appear to be the desired behavior here.